src/wdio/services/TerraService.js

```diff
@@ -65,12 +65,13 @@ const beAccessible = (options) => {
 /**
 * Generates a test for each themed property given and runs a screenshot assertion.
 * @property {Object} customProperties - An object containing the CSS custom properties to assert.
+* @property {Object} options - Additional options for testing.
 */
-const themeEachCustomProperty = (customProperties) => {
+const themeEachCustomProperty = (customProperties, options = {}) => {
```

Comment:
Is command is pretty straight forward. Do you thinking we need to allow an options object or could we directly pass the selector?

```
const shouldTheme = (customProperties, selector) => {
  const selectedElement = selector || '[data-reactroot]';
  Object.entries(customProperties).forEach(([key, value]) => {
    it(`themed [${key}]`, () => {
      browser.execute(`document.documentElement.style.setProperty('${key}', '${value}')`);
      expect(browser.checkElement(selectedElement)).to.matchReference();
    });
  });
};
```

Reply:
I kept it to options to add better passivity to it, and keep it consistent with the API the other functions have. If we ever had to add additional parameters to the method, that could make a potential non passive change, as you'd have to re arrange the arguments of the method.

Reply:
It might also be nice to allow an option to be passed in `wdio.conf.js` to set a default selector globally. This will prevent users from having to set this option everywhere.

Reply:
So I think this looks just fine, though I honestly, I would be more comfortable siting down and mapping out a wishlist of expected functions to be added to wdio before adding enhancements to the current functions. 

For instance, allowing this command to take options seems like overkill here. This is a very specific command and an options object adds an additional layer of complexity to its use that I don't think is needed here. Plus it lesses the readability of the command. 

As per @mhemesath suggestion regarding a global option, I also think that would be valuable, but I'd again want to scope out the use-case and future commands we wish to add. 

I think there are a lot of commands we can add to be useful which would then need a similar setup to this if we make changes now without thinking them through. 

@bjankord @mjhenkes @mhemesath  Thoughts??

Reply:
> This is a very specific command and an options object adds an additional layer of complexity to its use that I don't think is needed here

I tend to agree. I think we could get by with `Terra.should.themeEachCustomProperty('.my-selector', {})`; In this case its easy to check if the first param is a string, assume a custom selector is added, else use default.

Reply:
See if this works better. I do like the idea of the global setting. https://github.com/cerner/terra-toolkit/pull/48/commits/fadce63e19d9c69b398a1b8da014c3717a1f064b